docs/using-actuated.md

```diff
@@ -0,0 +1,39 @@
+# Using actuated
+
+[Actuated](https://actuated.dev/) is available as a GitHub App which can be
+enabled on repositories, and allows GH Actions to execute on runners hosted by
+the CNCF. [See the announcement](https://actuated.dev/blog/arm-ci-cncf-ampere)
+
```

Comment:
Can you add a few words when to use actuated and when not to use it and to stick with GH provided runners? I assume that it is not the idea to do ALL runs on actuated?

Reply:
I've done that in d05b91c0c1abfc945e9a18211a6e306c0f9ee183.

Reply:
thx!